app/api/entities.js

```diff
@@ -8,8 +8,9 @@ export default {
   // GET
   getByUris (uris, refresh, relatives) {
```

Comment:
what about giving that control to consumers instead?
```suggestion
  getByUris ({ uris, refresh, relatives, autocreate }) {
```

Reply:
yagni ? well see when well need it no ?